src/WCAdminHelper.php

```diff
@@ -46,4 +68,20 @@ public static function is_wc_admin_active_for( $seconds ) {
 
 		return ( $wc_admin_active_for >= $seconds );
 	}
+
+	/**
+	 * Test if WooCommerce Admin has been active within a pre-defined range.
+	 *
+	 * @param string $range range available in WC_ADMIN_STORE_AGE_RANGES.
+	 * @return bool Whether or not WooCommerce admin has been active within the range.
+	 */
+	public static function is_wc_admin_active_in_date_range( $range ) {
```

Comment:
Curious if there is a way to add the accepted string types to the `$range` method here, instead of any string?

Reply:
Not yet sadly, enums don't come until PHP 8.1 https://stitcher.io/blog/php-enums

In the meantime you could do a check like `array_key_exists( $range, self::WC_ADMIN_STORE_AGE_RANGES )` and throw an `\InvalidArgumentException` if it fails.

Reply:
Thanks that's a nice way as well (I look forward to enums :) ), fixed as part of this commit: https://github.com/woocommerce/woocommerce-admin/pull/6969/commits/aa2cb35f30506114f31f3adafa955b9e3885bea2